docs/tutorials/permissions.rst

```diff
@@ -247,12 +141,12 @@ We can add an article.
         }
     }
 
-Everybody can read the article:
+Indeed, using another user like *Natim*, we can read the article:
```

Comment:
Is the user string case-sensitive? It's capitalised here, but in the `--auth` it's not.